README.md

```diff
@@ -5,6 +5,26 @@ Generic encryption software
 
 ## Usage
 
+encryptio uses a CLI interface(using [argparse](https://docs.python.org/3/library/argparse.html)): 
+
+```bash
+python3 encryptio.py [-h] encryptionAlgorithm encryptionKey encryptionLibrary encryptionObjects (encryptionObjectOptions)
```

Comment:
hier müssen die encryptionObjectOptions vor den encryptionObjects kommen.. Und soll das nicht nur Einzahl sein?

Reply:
I changed the encryptionObject to encryptionObjectIdentifier and encryptionObjectOptions to encryptionObjects.